easybuild/easyconfigs/j/jax/jax-0.2.18-correct_libraries.patch

```diff
@@ -0,0 +1,63 @@
+diff -ruN jax_orig/build/build.py jax/build/build.py
```

Comment:
please add a short patch description on top

Reply:
done

Reply:
yeah, each time it generates it gets deleted :D ill add it when it's ready

Reply:
fixed in 54fb8efadb